zerver/tests/test_transfer.py

```diff
@@ -41,16 +41,16 @@ def test_transfer_avatars_to_s3(self) -> None:
         transfer_avatars_to_s3(1)
 
         path_id = user_avatar_path(user)
-        image_key = bucket.get_key(path_id)
-        original_image_key = bucket.get_key(path_id + ".original")
-        medium_image_key = bucket.get_key(path_id + "-medium.png")
+        image_key = bucket.Object(path_id)
+        original_image_key = bucket.Object(path_id + ".original")
+        medium_image_key = bucket.Object(path_id + "-medium.png")
 
-        self.assertEqual(len(bucket.get_all_keys()), 3)
-        self.assertEqual(image_key.get_contents_as_string(), open(avatar_disk_path(user), "rb").read())
-        self.assertEqual(original_image_key.get_contents_as_string(), open(avatar_disk_path(user, original=True), "rb").read())
-        self.assertEqual(medium_image_key.get_contents_as_string(), open(avatar_disk_path(user, medium=True), "rb").read())
+        self.assertEqual(sum(1 for _ in bucket.objects.all()), 3)
```

Comment:
`len(list(bucket.objects.all()))` is the right way to do this, rather than this weird `sum(1...` game.